.appveyor-mingw.yml

```diff
@@ -0,0 +1,18 @@
+version: 1.0.{build}

```

Comment:
how does appveyor know to run this yml file?  I looked at the CI builds attached to this PR and it doesn't appear to be running it.  is there an appveyor run that I can see executing the builds described by this file?